controller/scaling/worker/init

```diff
@@ -62,25 +60,24 @@ def scalling(s):
     _do(ssh, "sed '/saltstack/d' /etc/apt/sources.list")
     _do(ssh, 'echo "deb http://repo.saltstack.com/apt/debian/8/amd64/archive/2016.3.2 jessie main" >> /etc/apt/sources.list.d/saltstack.list')
     _do(ssh, 'apt-get update')
-    _do(ssh, 'apt-get install salt-minion=2016.3.2+ds-1 --yes')
+    _do(ssh, 'apt-get install salt-minion=2016.3.2+ds-1 salt-common=2016.3.2+ds-1 --yes')
     _do(ssh, 'easy_install --upgrade pip')
     _do(ssh, 'pip install docker-py pip --upgrade')
     _do(ssh, 'echo "%s" > /etc/salt/minion_id' % name)
     _do(ssh, "sed -i -e 's/VHG/%s/g' /etc/hostname" % name)
     _do(ssh, "sed -i -e 's/VHG/%s/g' /etc/hosts" % name)
     _do(ssh, '/etc/init.d/hostname.sh start')
-    _do(ssh, 'chmod +x /usr/local/bin/tnova_send_metric')
     # for everbody
     _do(ssh, "echo %s salt >> /etc/hosts" % masterIp)
     _do(ssh, 'echo "roles:" > /etc/salt/grains')
     _do(ssh, 'echo "  - core" >> /etc/salt/grains')
     _do(ssh, 'echo "  - worker" >> /etc/salt/grains')
     _do(ssh, "service salt-minion restart")
 
-    print "start salt (sleep 30s)"
-    time.sleep(30)
-    ssh.connect("127.0.0.1", username="root")
-    _do(ssh, "salt '%s' state.highstate" % name)
+    print "start salt"
+    _do(ssh, "salt-call state.highstate") # do only on this node because restart docker service (and kill current job)
```

Comment:
we use https://github.com/dngroup/t-nova/blob/master/salt/worker/file/docker but we also change /etc/default/docker https://github.com/dngroup/t-nova/blob/master/salt/worker/file/docker
when we re do the orch the formula remove ours  /etc/default/docker and restart docker to do the modification 
and after that we restart docker again to re do the modification.